circuit_knitting/cutting/automated_cut_finding.py

```diff
@@ -137,6 +137,8 @@ class OptimizationParameters:
     seed: int | None = OptimizationSettings().seed
     max_gamma: float = OptimizationSettings().max_gamma
     max_backjumps: None | int = OptimizationSettings().max_backjumps
+    gate_lo: bool = OptimizationSettings().gate_lo
+    wire_lo: bool = OptimizationSettings().wire_lo
```

Comment:
One might want to investigate what happens when one sets both `gate_lo` and `gate_lo` to `False`.  Ideally this will result in an informative error message (right?).

Reply:
Yes, for sure.

Reply:
Also, acknowledging that we'll probably change the name of `gate_lo` eventually. since I doubt it makes any sense to implement LOCC gate cutting now that https://arxiv.org/abs/2312.11638 is out.

Reply:
I have gone ahead and added some of the updates that we discussed ([7fcfcb1](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/586/commits/7fcfcb1c051d9f6879709d9186f95323c9ebf4af)).